src/func.d

```diff
@@ -3497,6 +3496,62 @@ public:
     }
 
     /***********************************************
+     * Check that the function contains any closure.
+     * If it's @nogc, report suitable errors.
+     * This is mostly consisten with FuncDeclaration::needsClosure().
```

Comment:
Being in D now, need to use Ddoc comments for functions. In particular, the return value is not documented with a `Returns:` block.